modules/nf-core/pretextmap/main.nf

```diff
@@ -0,0 +1,38 @@
+
+process PRETEXTMAP {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::pretextmap=0.1.9 bioconda::samtools=1.16.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-f3591ce8609c7b3b33e5715333200aa5c163aa61:c6242a6c1a522137de7a9e9ff90779ede11cf5c5-0':
+        'quay.io/biocontainers/mulled-v2-f3591ce8609c7b3b33e5715333200aa5c163aa61:c6242a6c1a522137de7a9e9ff90779ede11cf5c5-0' }"
+
+    input:
+    tuple val(meta), path(bam)
```

Comment:
PretextMap also takes `pairs` and `sam` file. Call it `input` instead. And describe the different options `sam/bam` or `pairs` in `meta.yml`

Reply:
done